sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroGenericRecordToStorageApiProto.java

```diff
@@ -172,7 +178,7 @@ public static TableSchema protoTableSchemaFromAvroSchema(Schema schema) {
    * @return A dynamic message representation of a Proto payload to be used for StorageWrite API
    */
   public static DynamicMessage messageFromGenericRecord(
-      Descriptor descriptor, GenericRecord record) {
+      Descriptor descriptor, GenericRecord record, @Nullable String changeType, long csn) {
```

Comment:
Update Javadoc with details on the two new parameters.

Reply:
Perhaps rename 'csn' to a more descriptive name.

Reply:
done